bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/ggh012j_5f_NLO_FXFX_M125/ggh012j_5f_NLO_FXFX_M125_customizecards.dat

```diff
@@ -0,0 +1,23 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 173.0
+#set param_card yukawa 6 173.0
+set param_card mass 25 125.0
```

Comment:
You could remove these lines since they are not needed (the X mass is already 125 GeV in your model)